mirgecom/eos.py

```diff
@@ -286,6 +335,46 @@ def __init__(self, pyrometheus_mech, temperature_guess=300.0):
         """
         self._pyrometheus_mech = pyrometheus_mech
         self._tguess = temperature_guess
+        self._transport_model = transport_model
+
+    def transport_model(self):
+        """Get the transport model object for this EOS."""
+        return self._transport_model
+
+    def heat_capacity_cp(self, cv: ConservedVars = None):
+        r"""Get mixture-averaged specific heat capcity at constant pressure.
+
+        Parameters
+        ----------
+        cv: :class:`mirgecom.fluid.ConservedVars`
+            :class:`mirgecom.fluid.ConservedVars` containing at least the mass
+            ($\rho$), energy ($\rho{E}$), momentum ($\rho\vec{V}$), and the vector of
+            species masses, ($\rho{Y}_\alpha$).
+        """
+        if cv is None:
+            raise ValueError("EOS.gamma requires ConservedVars (cv) argument.")
+        temp = self.temperature(cv)
+        y = cv.species_mass_fractions
```

Comment:
?

Reply:
Woops.  This utility property of the CV needed to be brought down from above.